src/pages/index.js

```diff
@@ -3,141 +3,201 @@ import "../pages/index.css";
-const profileModalName = document.querySelector("#modal-input-name");
-const profileModalDescription = document.querySelector(
-  "#modal-input-description"
-);
-const newCardModalTitle = document.querySelector("#modal-input-title");
-const newCardModalUrl = document.querySelector("#modal-Url");
-
-//Card Queries
-const addNewCardModal = document.querySelector("#add-new-card");
-const addNewCardForm = document.querySelector("#add-card-form");
-const addNewCardSubmit = document.querySelector("#new-card-submit");
-const cardList = document.querySelector(".cards__list");
-
-//VALIDATION
-const config = {
-  formSelector: ".modal__form",
-  inputSelector: ".modal__input",
-  submitButtonSelector: ".modal__button",
-  inactiveButtonClass: "modal__button_disabled",
-  inputErrorClass: "modal__input_type_error",
-  errorClass: "modal__error_visible",
-};
+});
+
 //SECTION RENDERER
 const cardSection = new Section(
   {
-    items: initialCards,
     renderer: generateCard,
   },
   ".cards__list"
 );
-cardSection.renderItems();
 
 function generateCard(item) {
   const cardElement = renderCard(item);
-  cardSection.addItem(cardElement.getCard());
+  cardSection.addItem(cardElement);
 }
 
 //RENDER CARD
 function renderCard(data) {
   const card = new Card(data, "#card-template", {
     handleImageClick: () => imagePopup.open(data),
+    handleDeleteClick,
+    handleCardLike
   });
-  return card;
+  return card.getCard();
 }
 
 //IMAGE POPUP
 const imagePopup = new PopupWithImage("#preview-image-modal");
 imagePopup.setEventListeners();
 
-//FORM POPUP
+//DELETE CARD POPUP
+const deleteCardPopup = new PopupWithConfirmation("#card__delete-modal");
+deleteCardPopup.setEventListeners();
+
+
+function handleDeleteClick(card) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    deleteCardPopup.setSubmitText(true, "Deleting...");
+    api
+      .deleteCard(card.id)
+      .then(() => {
+        deleteCardPopup.close();
+        card.removeCard();
+      })
+      .catch((err) => console.log(err))
+      .finally(() => deleteCardPopup.setSubmitText(false));
+  });
+}
+
 
+
+//FORM POPUP
 const openImagePopup = new PopupWithForm(
   "#add-new-card",
   handleImageFormSubmit
 );
-const editProfile = new PopupWithForm(
-  "#profile-edit-modal",
-  handleFormSubmit
+const editProfile = new PopupWithForm("#profile-edit-modal", handleFormSubmit);
+const profilePicEdit = new PopupWithForm(
+  "#profile__image-edit",
+  profilePicEditSubmit
 );
+
 editProfile.setEventListeners();
+profilePicEdit.setEventListeners();
 openImagePopup.setEventListeners();
 
-const editFormValidator = new FormValidator(config, profileModalForm);
-const cardFormValidator = new FormValidator(config, addNewCardModal);
+const handleEditFormSubmit = new FormValidator(config, profileModalForm);
+const handleProfileFormSubmit = new FormValidator(config, profilePicModal);
+const handleCardFormSubmit = new FormValidator(config, addNewCardModal);
```

Comment:
* You shouldn't have changed the names of the validators

These are validators (not submit handlers)


```js

const profileFormValidator = new FormValidator(config, profileModalForm);
const avatarFormValidator = new FormValidator(config, profilePicModal);
const cardFormValidator  = new FormValidator(config, addNewCardModal);
```